pnp.README.md

```diff
@@ -0,0 +1,3 @@
+some PnP notes (delete this file btw):
```

Comment:
It isn't hosted on Vercel so you can delete this file btw

Reply:
Oh ok